Templates/partials/run_header.jinja2

```diff
@@ -14,6 +14,11 @@
             ongoing
           {% endif %}
         </div>
+        {% if run.modifiers %}
+        <div class="modifiers">
+          <div title="{{ run.modifiers_with_desc }}">Modifiers</div>
```

Comment:
This creates a hover-only effect that's not visible on mobile.

I would instead suggest that we put this information in the Neow box to the left. It's empty on dailies anyway since we don't get a Neow bonus then. Then we'd have this information in `run_single.jinja2` rather than in the already kinda crammed header. Instead we could add an icon in the header, like we do when we're running a modded character.

Do you think you can fix those? Feel free to ping me if you need pointers!

Reply:
Personally, whenever I do custom runs, I run them with a mod that will force Neow Bonuses. I think it might be fine to add it in the box anyways though. I think it would be fine to put under the Neow Bonus info. I was thinking about an icon instead of straight text, but didn't find one on Fontawesome that I liked that much. Do you have one you would choose? If not I'll try to find an appropriate one

Reply:
Yeah, put it below it if it's available!

How about the [flask](https://fontawesome.com/icons/flask?s=solid&f=classic) icon? That usually signals experimentation and deviation from normalcy :eyes: